lib/ffmpeg/decoderbase.py

```diff
@@ -19,13 +19,18 @@ def before(self):
     super().before()
     self.refctx = []
     self.renderDevice = get_media().render_device
+    self.cmd = ffmpeg_cmd()
 
   @timefn("ffmpeg")
   def call_ffmpeg(self):
+    self.source = wsl_path_convert(self.source)
+    self.decoded = wsl_path_convert(self.decoded)
+    self.output_format = wsl_format_covert(self.hwaccel)
     return call(
-      (
-        "ffmpeg -hwaccel {hwaccel} -init_hw_device {hwaccel}=hw:{renderDevice}"
-        " -hwaccel_output_format {hwaccel} -hwaccel_flags allow_profile_mismatch"
+      ("{cmd} -hwaccel {hwaccel}").format(**vars(self)) + (
+        " -init_hw_device {hwaccel}" + wsl_device("=hw:{renderDevice}")
+      ).format(**vars(self)) + (
+        " -hwaccel_output_format {output_format} -hwaccel_flags allow_profile_mismatch"
```

Comment:
I think we can apply this technique to all plugins and OS by setting it to `hwformat`... which means we can also remove the "hwdownload" filter.  But, I'd like to test it first.

So, for now, let's avoid this `{output_format}` change for now... it works the other way just fine.

Reply:
ah, d3d11va does not work, only d3d11 does.  So I suggest we allow subclass to override with new variable.